homeassistant/components/bmw_connected_drive/binary_sensor.py

```diff
@@ -236,26 +231,30 @@ class BMWConnectedDriveSensor(BMWConnectedDriveBaseEntity, BinarySensorEntity):
 
     def __init__(
         self,
-        account: BMWConnectedDriveAccount,
+        coordinator: BMWDataUpdateCoordinator,
         vehicle: ConnectedDriveVehicle,
         description: BMWBinarySensorEntityDescription,
         unit_system: UnitSystem,
     ) -> None:
         """Initialize sensor."""
-        super().__init__(account, vehicle)
```

Comment:
Override `def _handle_coordinator_update(self) -> None:` instead so you don't have to convert back to using the long hand `@property` method

Reply:
Adjusted.